quiz.css

```diff
@@ -0,0 +1,69 @@
+/*Block Styling and Positioning*/
+.medium,
+.dark,
+.tshirt,
+.green {
+  border:1px solid lightblue;
```

Comment:
It looks like you're giving each article a unique class. One of the powers of classes is that you can have multiple elements with the same class and apply the same styles to all of them. I you have something you want to be unique, typically you would use an ID